ducktape/__main__.py

```diff
@@ -0,0 +1,131 @@
+from ducktape.tests.test import TestLoader
```

Comment:
For some reason the PEP style guide recommends newlines for each:
https://www.python.org/dev/peps/pep-0008/#imports